phpstan.neon.dist

```diff
@@ -13,18 +13,7 @@ parameters:
 		- plugins/webp-uploads/load.php
 	scanDirectories:
 		- vendor/wp-phpunit/wp-phpunit/
+	scanFiles:
+		- bin/phpstan/functions.stub
 	dynamicConstantNames:
 		- PERFLAB_OBJECT_CACHE_DROPIN_VERSION
-	ignoreErrors:
-		# False positive as WordPress ships with a polyfill.
-		-
-			message: "#^Function str_starts_with not found\\.$#"
-			paths:
-				- load.php
-				- plugins/optimization-detective/optimization.php
-				- plugins/speculation-rules/class-plsr-url-pattern-prefixer.php
-		-
-			message: "#^Function str_contains not found\\.$#"
-			paths:
-				- plugins/auto-sizes/hooks.php
-				- plugins/dominant-color-images/hooks.php
```

Comment:
I think we should keep this, but use `/*` to ignore these rules globally, rather than copying the shims to our repo.

Reply:
So like what was done in https://github.com/WordPress/performance/pull/1188/commits/51d1894cf4cc7c2b59c7f293058b722b4870c0fa?

Reply:
I think I have a better solution. See 828c87fe